tests/routes/semrush-route-test.php

```diff
@@ -108,6 +135,26 @@ public function test_is_valid_code_with_valid_code_given() {
 		$this->assertTrue( $this->instance->has_valid_code( '123456' ) );
 	}
 
+	/**
+	 * Tests the country code is valid, with invalid country code given as input.
+	 *
+	 * @covers ::has_valid_country_code
+	 */
+	public function test_is_valid_country_code_with_invalid_code_given() {
+		$this->assertFalse( $this->instance->has_valid_country_code( '' ) );
+		$this->assertFalse( $this->instance->has_valid_country_code( 'abc' ) );
+		$this->assertFalse( $this->instance->has_valid_country_code( '12' ) );
```

Comment:
See previous comment: the country code is a hardcoded value: are there actual cases where users can select and submit an empty string or a numeric value? See also comment on the SEMrush documentation, where some country codes are prepended with `mobile-`

Reply:
Users won't select any non-valid value as long as they use the Country Selector, but we shouldn't trust input since anyone with authorization would be able to POST a wrong value that has effects for any other user.
See above for `mobile-*`.